test/role.e2e-spec.ts

```diff
@@ -0,0 +1,117 @@
+import { INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { AppModule } from '../src/app.module';
+import { database } from './constants';
+import * as mongoose from 'mongoose';
+import * as request from 'supertest';
+import { RoleType } from '../src/role/enum/role.enum';
+import { CreateUserDto } from '../src/user/dto/create-user.dto';
+
+beforeAll(async () => {
+  await mongoose.connect(database);
+  await mongoose.connection.db.dropDatabase();
+});
+
+afterAll(async (done) => {
+  await mongoose.disconnect(done);
+});
```

Comment:
Warto na koniec dodać `await mongoose.connection.db.dropDatabase();`
bez tego ostatni test który będzie wykonywany zostawi ślad w bazie danych i moze powodować błąd

Reply:
Aaaa widzisz, to to załatwia sprawę, dobrze wiedzieć, wczoraj mi się właśnie testy usera sypały bo zostawiał ślad, ale ogarnełam to tak że na koniec testów usuwałam to :D _

Reply:
ja własnie jak mi napisałas ze te moje testy nie przechodzą to tez tam troche pozmieniałem bo okazało się ze to beforall i after all one muszą być w srodku tego descriibe, to tez pamietaj o tym

Reply:
oki, zmieniłam, dzięki!